modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/OperationMgtService.java

```diff
@@ -131,8 +139,8 @@ public Response configureDeviceLock(@HeaderParam("Accept") String acceptHeader,
     @POST
     @Produces(MediaType.APPLICATION_JSON)
```

Comment:
While we allow any type of response via the accept header,  why do we also limit response to JSON?